spec/build/env_spec.rb

```diff
@@ -101,13 +103,31 @@
         expect(keys).to_not include('FOOMASTER')
       end
       it 'includes values restricted to foo-(dev) branch' do
-        expect(vars.find {|var| var.key == 'MULTIBRANCHVARIABLE'}.value).to eq('foodevvalue')
+        expect(vars.select {|var| var.key == 'MULTIBRANCHVARIABLE'}.last.value).to eq('foodevvalue')
```

Comment:
We want to test that the _last_ one that sets `MULTIBRANCHVARIABLE` has the right value, not the _first_ one returned by `#find`.